express/styles/styles.css

```diff
@@ -339,6 +339,36 @@ a.button.dark.reverse:any-link:focus {
   color: var(--color-info-accent-hover);
 }
 
+a.button.gradient:any-link,
+a.button.gradient:any-link:hover,
+a.button.gradient:any-link:active,
+a.button.gradient:any-link:focus {
+  color: var(--color-white);
+  border-color: transparent;
+  background: linear-gradient(320deg, #7C84F3, #FF4DD2, #FF993B, #FF4DD2, #7C84F3, #FF4DD2,#FF993B );
```

Comment:
Should we store this in a var?

Reply:
Up to you, not sure if we will use it again...